Casks/n/northern-softworks-cache-cleaner.rb

```diff
@@ -1,20 +1,52 @@
 cask "northern-softworks-cache-cleaner" do
-  version "20.0"
   sha256 :no_check
 
-  # Homepage, livecheck regex, and app change with major macOS releases
-
-  url "https://www.northernsoftworks.com/downloads/nscc.dmg"
   name "Northern Softworks Cache Cleaner"
   desc "General purpose system maintenance tool"
-  homepage "https://www.northernsoftworks.com/sequoiacachecleaner.html"
+  homepage "https://www.northernsoftworks.com/cachecleanerfeatures.html"
+
+  on_mojave do
```

Comment:
```suggestion
  on_mojave :or_older do
```

We generally set the older version to `or_older`, and then if need be add a global `depends_on macos...`